lib/httpserver/httpserver.go

```diff
@@ -374,8 +376,10 @@ func CheckBasicAuth(w http.ResponseWriter, r *http.Request) bool {
 	}
 	username, password, ok := r.BasicAuth()
 	if ok && username == *httpAuthUsername && password == *httpAuthPassword {
+		successfulAuthAtemptsTotal.Inc()
 		return true
 	}
+	unSuccessfulAuthAtemptsTotal.Inc()
```

Comment:
That's not always unsuccessful attempt. If I'm not mistaken, browser does first request as is without creds. If server responds with header `WWW-Authenticate`, then browser suggests to user to enter the creds. 
From the user perspective this happens immediately. But with this metric increment you'll be getting false positives.